pkg/kubelet/images/pullmanager/fs_pullrecords.go

```diff
@@ -181,10 +181,52 @@ func (f *fsPullRecordsAccessor) DeleteImagePulledRecord(imageRef string) error {
 	return err
 }
 
+func (f *fsPullRecordsAccessor) intentsSize() (uint, error) {
+	intentsCount, err := countCacheFiles(f.pullingDir)
+	if err != nil {
+		return 0, err
+	}
+	return intentsCount, nil
+}
+
+func (f *fsPullRecordsAccessor) pulledRecordsSize() (uint, error) {
+	pulledRecordsCount, err := countCacheFiles(f.pulledDir)
+	if err != nil {
+		return 0, err
+	}
+	return pulledRecordsCount, nil
+}
+
+func countCacheFiles(dir string) (uint, error) {
+	var cacheFilesCount uint
+	entries, err := os.ReadDir(dir)
```

Comment:
I'm curious how many cache files there might be? It might be better to iterate over the directory with the (*File) ReadDir(n int) to keep memory usage at a minimum.